packages/sdk-components-react-remix/src/webhook-form.tsx

```diff
@@ -55,8 +100,28 @@ export const WebhookForm = forwardRef<
       onStateChange?.(state);
     });
 
+    /**
+     * Add hidden field generated using js with simple jsdom detector.
+     * This is used to protect form submission against very simple bots.
+     */
+    const handleSubmitAndAddHiddenJsField = (
+      event: React.FormEvent<HTMLFormElement>
+    ) => {
+      const hiddenInput = document.createElement("input");
+      hiddenInput.type = "hidden";
+      hiddenInput.name = formBotFieldName;
+      hiddenInput.value = isJSDom() ? "jsdom" : Date.now().toString(16);
```

Comment:
I couldn't find where we treat this jsdom value

Reply:
Ok, wouldn't guess it and would assume the value is used somehow, needs at least a comment